Content.Server/Nutrition/EntitySystems/FoodSystem.cs

```diff
@@ -421,6 +446,39 @@ private void OnForceFeedCancelled(ForceFeedCancelledEvent args)
         {
             args.Food.InUse = false;
         }
+
+        /// <summary>
+        ///     Is an entity's mouth accessible, or is it blocked by something like a mask? Does not actually check if
+        ///     the user has a mouth. Body system when?
+        /// </summary>
+        public bool IsMouthBlocked(EntityUid uid, [NotNullWhen(true)] out EntityUid? blockingEntity,
+            InventoryComponent? inventory = null)
+        {
+            blockingEntity = null;
+
+            if (!Resolve(uid, ref inventory))
+                return false;
```

Comment:
Given we might want something like an animal to eat which may not have an inventory probably don't log on these.

Reply:
Ah yep. I keep forgetting resolve will log missing resolves. Fixed now.